.github/workflows/ci.yml

```diff
@@ -1,6 +1,11 @@
----
 name: CI
-on: [push, pull_request]
+
+on: 
```

Comment:
So did they fix it? A couple of years ago, this syntax was not valid (https://github.community/t/how-to-trigger-an-action-on-push-or-pull-request-but-not-both/16662/5), and I was not aware that they fixed it

Reply:
yeah this is what we use everywhere to avoid running CI multiple times on PRs